code/datums/mind/antag.dm

```diff
@@ -44,7 +44,7 @@
 
 /datum/mind/proc/has_antag_datum(datum_type, check_subtypes = TRUE)
 	if(!datum_type)
-		return
+		return FALSE
```

Comment:
Why change this? Seems like this should never trigger. 

In any event this proc returns `datum | null` so making it return `FALSE` is a bit silly

Reply:
deprecated change